log4j-core/src/main/java/org/apache/logging/log4j/core/pattern/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Provides classes implementing format specifiers in conversion patterns.
  */
 @Export
-@Version("2.21.1")
+@Version("2.22.0")
```

Comment:
@ppkarwasz, you're right. I had blindly applied the `bnd-baseline:check` suggetion. Replacing it with `2.24.0`:

```suggestion
@Version("2.24.0")
```